aioipfs/api.py

```diff
@@ -1004,16 +1009,29 @@ async def peers(self, topic=None):
         return await self.fetch_json(
             self.url('pubsub/peers'), params=params)
 
-    async def pub(self, topic, data):
+    async def pub(self, topic, data: Optional[str]):
         """
         Publish a message to a given pubsub topic.
 
         :param str topic: topic to publish the message to
         :param str data: message data
         """
 
-        return await self.post(self.url('pubsub/pub'),
-                               None, params=quote_args(topic, data))
+        params = {
+            ARG_PARAM: multibase.encode(
```

Comment:
Why not using `base64.urlsafe_b64encode` ?

Reply:
Same encoding in the end, right? I went straight to the point: the new spec says multibase, I used the multibase package.

Reply:
I see.